src/lib/viewers/image/ImageControls.tsx

```diff
@@ -22,19 +33,30 @@ export default function ImageControls({
     scale,
 }: Props): JSX.Element {
     return (
-        <ControlsBar>
-            <ZoomControls onZoomIn={onZoomIn} onZoomOut={onZoomOut} scale={scale} />
-            <RotateControl onRotateLeft={onRotateLeft} />
-            <FullscreenToggle onFullscreenToggle={onFullscreenToggle} />
-            <AnnotationsControls
-                annotationColor={annotationColor}
-                annotationMode={annotationMode}
-                hasDrawing={hasDrawing}
-                hasHighlight={hasHighlight}
-                hasRegion={hasRegion}
-                onAnnotationModeClick={onAnnotationModeClick}
-                onAnnotationModeEscape={onAnnotationModeEscape}
-            />
-        </ControlsBar>
+        <>
+            <ControlsBar>
+                <ZoomControls onZoomIn={onZoomIn} onZoomOut={onZoomOut} scale={scale} />
+                <RotateControl onRotateLeft={onRotateLeft} />
+                <FullscreenToggle onFullscreenToggle={onFullscreenToggle} />
+                <AnnotationsControls
+                    annotationColor={annotationColor}
+                    annotationMode={annotationMode}
+                    hasDrawing={hasDrawing}
+                    hasHighlight={hasHighlight}
+                    hasRegion={hasRegion}
+                    onAnnotationModeClick={onAnnotationModeClick}
+                    onAnnotationModeEscape={onAnnotationModeEscape}
+                />
+            </ControlsBar>
+            {hasDrawing && annotationMode === AnnotationMode.DRAWING && (
```

Comment:
Rather than duplicating this in each set of controls, we could make a couple changes:

1. Update `ControlsBar` to count its children and return null if their length is zero
1. Create a simple wrapper around ColorPickerControl that checks the annotationMode and returns null if it's not "drawing"

Reply:
Sounds good, will implement those changes.